main.md

```diff
@@ -156,3 +161,5 @@ Note: `< >` should not be included in commit message. Example, `git fetch upstre
 `$ git diff --cached <commit-id>`
 - To check difference between changed staged for the next commit relative to a particular commit.
 
+### Help and Documentation
+`$ git help <command>`
```

Comment:
This section can be the first section. Please put this at top before all other sections